### PR TITLE
Made ggev functions as non-scalar functions and support for lapack-src v0.13

### PR DESCRIPTION
**Pull request by selvavm**
Made ggev functions as non-scalar functions and support for lapack-src v0.13

#27

### CODE REVIEW

**Comment by selvavm** [comment]
I have also changed to argument case to lower. Check if it is okay? Other alternative is to ignore case

**Comment by selvavm** [comment]
@IvanUkhov - I was facing issue with `lsame` and added a code similar to `f.name=='lsame'`. Is there better way to handle this?

File: bin/generate.py [hunk 1]
```diff
@@ -190,15 +201,15 @@ def format_body(f):
 def format_header_arguments(f):
     s = []
     for arg in f.args:
-        s.append('{}: {}'.format(arg[0], translate_argument(*arg, f=f)))
+        s.append('{}: {}'.format(arg[0].lower(), translate_argument(*arg, f=f)))
```
Comment (IvanUkhov) [inline bin/generate.py:204]: Why is this needed?

**Comment by IvanUkhov** [comment]
I propose to just skip `lsame`. Not sure if anybody would want to call it. Please pull and take a look.

File: bin/generate.py (hunk 1 above)
Comment (selvavm) [reply bin/generate.py:204]: I added tolower for all the arguments because there were too many warnings. I am also not sure if this is needed here as well. 

If you dislike the idea of tolower, then we should add the ignore.

Comment (IvanUkhov) [reply bin/generate.py:204]: This is a good thing to lowercase. In this high-level crate, we want to see Rust-friendly names.

It actually made me realize that the logic in `generate.py` is broken, as it assumes everything is lowercase. I’ll push a fix.

**Comment by IvanUkhov** [comment]
It’s getting too big. I’ll merge this, and any other problems will be solved in separate pull requests.

**Comment by selvavm** [comment]
Thanks for merging.

File: bin/generate.py (hunk 1 above)
Comment (selvavm) [reply bin/generate.py]: Thanks. And let me know if you need me to change something. Waiting for your publish to crates.io

Comment (IvanUkhov) [reply bin/generate.py]: Done.
